.kokoro

```diff
@@ -74,6 +74,12 @@ run_bazel() {
 
   fix_bazel_imports
 
+  if [ -n "$KOKORO_BUILD_NUMBER" ]; then
+    bazel version
+    use_bazel.sh 0.11.0
```

Comment:
Do we prefer this over `use_bazel.sh latest`?

Reply:
Ah, good catch. `use_bazel.sh latest` is probably better.